include/deal.II/matrix_free/operators.h

```diff
@@ -30,6 +30,162 @@ DEAL_II_NAMESPACE_OPEN
 namespace MatrixFreeOperators
 {
   /**
+   * Base class for matrix-free operators which can be used both at the finest
+   * mesh or at a certain level in geometric multigrid.
+   */
+  template <int dim, int fe_degree, int n_q_points_1d = fe_degree+1, typename number = double>
```

Comment:
Any reason why you put the templates `fe_degree` and `n_q_points_1d` on this class? You don't seem to use them. Even though there might be good reasons to use them in a derived class because `FEEvaluation` requires templates, I do not see why we would want to have them in the base class.